static_timing/build_defs.bzl

```diff
@@ -41,13 +42,14 @@ def _run_opensta_impl(ctx):
         "NETLIST": netlist.path,
         "TOP": synth_info.top_module,
         "LOGFILE": sta_log.path,
-        "LIBERTY": liberty_file.path,
-        "TCL_LIBRARY": opensta_runfiles_dir + "/tk_tcl/library",
+        "LIBERTY": default_liberty_file.path,
+        "ADDITIONAL_LIBERTIES": ",".join([f.path for f in additional_liberty_files]),
```

Comment:
Should we dedup with to make sure that they do not include LIBERTY?

Reply:
added